packages/grid/test/visual/grid.common.js

```diff
@@ -78,6 +78,39 @@ describe('grid', () => {
         });
       });
 
+      describe('column group with footer only', () => {
```

Comment:
The tests pass even without the CSS change.

Reply:
As discussed internally, the test was ineffective, probably because the diff algorithm doesn't notice the difference. Decided to skip tests for now.